src/library/rdt/src/rdt.c

```diff
@@ -39,6 +39,8 @@ SEXP Rdt(SEXP tracer, SEXP rho, SEXP options) {
         handler = setup_promise_tracing(options);
     } else if (!strcmp("debug", sys)) {
         handler = setup_debug_tracing(options);
+    } else if (!strcmp("specialsxp", sys)) {
```

Comment:
Yes, this is a good idea. It was implemented this way by you I think. :) We just haven't changed it yet.

I imagine the way to address this would be to call `.Call` with the desired function name from C or move this part of initialization to R.